### ISSUE TITLE
arm64 for Windows Releases

### ISSUE THREAD

**Issue by rjmayott** [pulsar-edit/pulsar#1031]
### Have you checked for existing feature requests?

- [X] Completed

### Summary

Please add support for arm64 processors in the official releases of Pulsar

### What benefits does this feature provide?

Native support for arm64 Windows builds

### Any alternatives?

Emulation of the current code

**Comment by Daeraxa** [comment]
I don't think we would have any objections other than we don't have anyone on the core team with such a device (typically all of our "official" releases are done on systems that somebody has handy to test with).

The main issue is the GitHub runners we use, since we moved to them from our CirrusCI platform we now use them for all the main CI and binary builds but we still need to use the old platform for two not supported - Linux ARM and macOS ARM (Mx chips). I'm not 100% sure if CirrusCI even supports Windows ARM containers/VMs.

However there was this announcement from GitHub recently - https://github.blog/2024-06-03-arm64-on-github-actions-powering-faster-more-efficient-build-systems/ - that implies that we *may* have access to both Linux and Windows ARM runners by the end of the year.

https://github.blog/2024-06-03-arm64-on-github-actions-powering-faster-more-efficient-build-systems/

> GitHub is ecstatic to unveil ArmⓇ-based Linux and Windows runners for GitHub Actions are now in Public Beta

> These runners are available to our customers on our GitHub Team and Enterprise Cloud plans. We expect to begin offering Arm runners for open source projects by the end of the year.

**Comment by rjmayott** [comment]
I have an X Elite powered device I would be happy to help test/debug on if we get to that point!
